### DISCUSSION TITLE
realtime output to --junitxml file

### DISCUSSION THREAD

**Discussion by wfast01** [pytest-dev/pytest#11621] [Q&A]
I am running pytest inside of Jenkins, and using the --junitxml option to view test results in Jenkins.  

I would like to use Jenkins' 'Junit Realtime Test Reporter' (https://www.jenkins.io/doc/pipeline/steps/junit-realtime-test-reporter/) in place of junit, so I can monitor real time test progress from Jenkins.  The output file from --junitxml is only created at the end of the test run.  

Is there a configuration that forces pytest output to the --junitxml file on a test-by-test basis?

**Comment by RonnyPfannschmidt** [comment]
Nobody implemented this so far

**Comment by wfast01** [comment]
Thanks for checking.
